src/main/java/cz/muni/fi/pa165/dao/HallDaoImpl.java

```diff
@@ -0,0 +1,41 @@
+package cz.muni.fi.pa165.dao;
+
+import java.util.List;
+import javax.persistence.*;
+import javax.inject.Named;
+import org.springframework.stereotype.Repository;
+
+import cz.muni.fi.pa165.entity.Hall;
+
+
+/**
+ * Interface of DAO for Hall entity

```

Comment:
Ten komentář měl asi být implementation of interface, ne?

Reply:
ano, upraveno